db/schema.rb

```diff
@@ -265,6 +266,7 @@
     t.string "provided_by_educator_name"
     t.integer "service_upload_id"
     t.datetime "estimated_end_date"
+    t.datetime "discontinued_at"
```

Comment:
Hey @Jngai! I was expecting to see one more field here, something like `discontinued_by_educator_id`. That's because we want to make sure we're storing a **complete** history of what happens to a service, including which educator discontinued the service. A service could be created by one educator and then discontinued by a second educator.